BrainPortal/app/models/background_activity/archive_task_workdir.rb

```diff
@@ -0,0 +1,47 @@
+
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2024
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# Archive a CBRAIN task.
+#
+# Options: archive_data_provider_id.
+#
+# Must be run on a Bourreau only.
+class BackgroundActivity::ArchiveTaskWorkdir < BackgroundActivity::TerminateTask
+
+  Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
+
+  def process(item)
```

Comment:
often no docs

Reply:
Yes, I plan to add this later when I finish the full integration on the user-side